src/QtImageReader.cpp

```diff
@@ -51,9 +60,70 @@ void QtImageReader::Open()
 	// Open reader if not already open
 	if (!is_open)
 	{
+
+#if USE_RESVG == 1
+		image = std::shared_ptr<QImage>(new QImage());
+		bool success;
+
+		QFileInfo image_file_info(QString::fromStdString(path));
+
+		// Only use resvg for files with a '.svg' extension
+		if (image_file_info.suffix() == "svg") {
+			// Init resvg structs
+			resvg_render_tree *rtree = NULL;
```

Comment:
nullptr

Reply:
Yes, initialized to a known value.  Is this causing a warning or error when you try to compile the code?

Reply:
Nah, just another best practice thing.

Reply:
Oh, I see!  The comment was a little too laconic and I didn't realize what @RichiCoder1  was saying.  I've updated the source, tested it and committed an update.